Extensions/newRelease.js

```diff
@@ -317,7 +318,7 @@
         LIST.update(BUTTON.isUnlistenedOnly())
         LIST.update(BUTTON.isPodcastOnly())
         BUTTON.update(LIST.getUnlistenedLen())
-        if (LIST.getLen() === 0) {
+        if (LIST.getLen() === 0 || (BUTTON.isUnlistenedOnly() && LIST.getUnlistenedLen() === 0)) {
```

Comment:
Fixes an issue where the menu message wasn't being shown if there existed new releases, but had already been listened